README.md

```diff
@@ -19,7 +19,8 @@ This generator uses Go's `crypto/rand` which implements a cryptographically secu
 
 ```
 cd ~
-GO111MODULE=on go get -u github.com/iwittkau/minio-keygen
+GO111MODULE=on
+go get -u github.com/iwittkau/minio-keygen
```

Comment:
```suggestion
go install github.com/iwittkau/minio-keygen@latest
```

... and remove `GO111MODULE=on`, please. [As of Go 1.16 this should work](https://golang.org/ref/mod#go-install).